helm-charts/charts/kube-starrocks/templates/init-pwd/configmap.yaml

```diff
@@ -1,4 +1,12 @@
-{{- if and .Values.initPassword.enabled .Release.IsInstall }}
+{{- if .Values.initPassword.enabled }}
+apiVersion: v1
+kind: Secret
```

Comment:
move the Secret from `job.yaml` to `configmap.yaml`, so secret will not be deleted when upgrade?